ext/debian/pxp-agent.default

```diff
@@ -0,0 +1,5 @@
+# Defaults for pxp-agent - sourced by /etc/init.d/pxp-agent
+
+# Startup options
+#DAEMON_OPTS=""
```

Comment:
DAEMON_OPTS is called PXP_AGENT_OPTS in the el init script. We should probably pick one.